### DISCUSSION TITLE
Updated to 17 finally and I have an error when using TTS (still works but NR shows it needs fixing.

### DISCUSSION THREAD

**Discussion by monsieurlatte** [zachowj/node-red-contrib-home-assistant-websocket#1619] [Q&A] [answered]
It says this for my homepods.  Entity ID 'media_player.jrs_bedroom' should be in the data property, not the target property, for the selected action.

All I have in the JSON is this for one of my tests

{
    "language": "en-GB",
    "message": "hello james"
    
}

**Comment by monsieurlatte** [comment]
After hours of research I stumbled up the insert test in NR that gave me an example to work with.  Der.

**Reply by inguy24** [reply]
what did you find? After updating I am encountering the same error on all of my tts.cloud_say actions as you indicated above as the targets have been somehow eliminated.

**Reply by zachowj** [reply]
@inguy24 the `entity_id` needs to be in the data field not the targets field.

**Reply by monsieurlatte** [reply, marked as answer]
> what did you find? After updating I am encountering the same error on all of my tts.cloud_say actions as you indicated above as the targets have been somehow eliminated.

I should have added to my post, you're right.  So originally I had this.
{
    "language": "en-GB",
    "message": "hello james"
    
}

and had to add the first line with the entity to fix it. 

{
    "entity_id":"media_player.jrs_bedroom",
    "language": "en-GB",
    "message": "hello james"
    
}
